src/inverseRobot.js

```diff
@@ -7,7 +7,40 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotKeys = Object.values(robot);
+  const robotValues = Object.keys(robot);
+  const inversedRobot = {};
+
+  for (let i = 0; i < robotKeys.length; i++) {
```

Comment:
do this with one loop
you can take value without 2 loops
like
inversedRobot[robot[key]]
```suggestion
  for (const key in robot) {
```

Reply:
2 loops it is very bad algorithm